src/components/test-slot/test-outcome/__tests__/test-outcome.spec.ts

```diff
@@ -109,13 +112,13 @@ describe('Test Outcome', () => {
 
         expect(store$.dispatch).toHaveBeenCalledWith(StartTest(component.slotDetail.slotId, component.category));
       });
-      it('should dispatch a start test action with the slot', () => {
-        component.slotDetail = testSlotDetail;
-        component.category = TestCategory.C;
-        component.startTest();
-
-        expect(store$.dispatch).toHaveBeenCalledWith(StartTest(component.slotDetail.slotId, component.category));
-      });
+      // it('should dispatch a start test action with the slot', () => {
+      //   component.slotDetail = testSlotDetail;
+      //   component.category = TestCategory.C;
+      //   component.startTest();
+      //
+      //   expect(store$.dispatch).toHaveBeenCalledWith(StartTest(component.slotDetail.slotId, component.category));
+      // });
```

Comment:
any reason this has been commented out?

Reply:
yes. will add a todo

Reply:
category wasnt being passed in from parent when i wrote that. its now dynamic from the state again. so this has now been reinstated